eng/CredScanSuppression.json

```diff
@@ -59,7 +61,9 @@
                 "sdk/identity/Azure.Identity/tests/Data/mock-arc-mi-key.key",
                 "sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/PemReaderTests.Keys.cs",
                 "sdk/storage/Azure.Storage.Common/tests/Shared/azurite_cert.pem",
-                "sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/tests/cert.pem"
+                "sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/tests/cert.pem",
+                "sdk/storage/Azure.Storage.Common/tests/Shared/AzuriteFixture.cs",
+                "sdk/mixedreality/Azure.MixedReality.Authentication/tests/Azure.MixedReality.Authentication.Tests/SessionRecords/MixedRealityStsClientLiveTests/GetToken"
```

Comment:
Same for this file

Reply:
oh I misunderstood this section - I thought you had to write the name of the file the secret was in, I'll revert